app/femr/business/services/system/EncounterService.java

```diff
@@ -89,7 +89,10 @@ public EncounterService(IMissionTripService missionTripService,
                     .where()
                     .eq("email", patientEncounterItem.getNurseEmailAddress());
 
-            IUser nurseUser = userRepository.findOne(nurseQuery);
+            ExpressionList<User> nurseQuery1 = QueryProvider.getUserQuery()
+                    .where()
+                    .eq("email", patientEncounterItem.getNurseFullName()); //Andrew Change
+            IUser nurseUser = userRepository.findOne(nurseQuery1);
```

Comment:
I don't think we want this change. When testing I got a null pointer exception. I think we need to delete nurseQuery1 and go back to nurseQuery

We only know the nurse user's email address at this point, so we need to use that in the query to get the User model.